src/main/java/com/oltpbenchmark/api/Worker.java

```diff
@@ -456,13 +456,13 @@ protected final void doWork(DatabaseType databaseType, TransactionType transacti
                         conn.rollback();
 
                     if (isRetryable(ex)) {
-                        LOG.debug(String.format("Retryable SQLException occurred during [%s]... current retry attempt [%d], max retry attempts [%d], sql state [%s], error code [%d].", transactionType, retryCount, maxRetryCount, ex.getSQLState(), ex.getErrorCode()), ex);
+                        LOG.debug(String.format("Retryable SQLException occurred during [%s]... current retry attempt [%d], max retry attempts [%d], sql state [%s], error code [%d].", transactionType, retryCount, maxRetryCount, ex.getSQLState(), ex.getErrorCode()));
```

Comment:
is this message getting dumped? This is a debug level message and logs should not be dumped by default. If it is not being dumped, please dont change it

Reply:
this is not getting dumped. Removed this change.